EXILED/Exiled.API/Enums/DamageType.cs

```diff
@@ -120,6 +120,11 @@ public enum DamageType
         /// </summary>
         SeveredHands,
 
+        /// <summary>
+        /// Damage caused by severed eyes.
+        /// </summary>
+        SeveredEyes,
+
```

Comment:
Replace the Scp1344 with this one

Reply:
Double check me

Reply:
Sorry i meaned than Scp1344 damage is not more used so replace the name of damagetype with the new one you made

Reply:
So good?

Reply:
Just removed line 261 no ?

Reply:
Done